runflake8

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+flake8 --max-line-length=110 \
+       --exclude=./docs/,./build/,./setup.py,./django_netjsongraph/migrations/ || exit 1
```

Comment:
Yeah. I understand. @nemesisdesign: Should I exclude the build files too? I get the lint errors in my locally set up application but they wouldn't pop up in Travis as build files are git ignored.